docs/tutorials/web/deploy.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Deploying ONNX Runtime Web
+description: Deploying ONNX Runtime Web
+parent: Web
+grand_parent: Tutorials
+has_children: false
+nav_order: 5
+---
+
+{::options toc_levels="2..4" /}
+
+# Deploying ONNX Runtime Web
```

Comment:
WebGPU is only accessible via secure context. Please refer point 3 in this document: https://developer.chrome.com/docs/web-platform/webgpu/troubleshooting-tips